openpbs.spec.in

```diff
@@ -249,12 +258,14 @@ utilization & efficiency, and simplify administration for
 HPC clusters, clouds and supercomputers.
 
 This package is intended for a client host and provides
-the PBS Professional user commands.
+the OpenPBS user commands.
 
 
 %package %{pbs_devel}
-Summary: PBS Professional Development Package
+Summary: OpenPBS Development Package
 Group: Development/System
+Conflicts: pbspro-devel
+Conflicts: pbspro-devel-ohpc
 
 %description %{pbs_devel}
 PBS Professional® is a fast, powerful workload manager and
```

Comment:
It still says PBS Professional here

Reply:
will update here as well as other places too..

Reply:
This would also be a good time to remove all unicode characters. Some distros have problems with them. That is why they have all been removed from the OpenHPC spec file.